mlflow/store/tracking/rest_store.py

```diff
@@ -541,6 +553,42 @@ def record_logged_model(self, run_id, mlflow_model):
         )
         self._call_endpoint(LogModel, req_body)
 
+    def create_logged_model(
+        self,
+        experiment_id: str,
+        name: str,
+        run_id: Optional[str] = None,
```

Comment:
```suggestion
        source_run_id: Optional[str] = None,
```
Shall we call it `source_run_id`? I'm thinking that adding the `source_` prefix prevents the user to assume that the logged_model object is **owned** by the run.

Reply:
Sounds good!

Reply:
@dbczumar what do you think?

Reply:
Yep! `source_run_id` is a good name here, same for the entity